### ISSUE TITLE
 [IOS] Unable to determine our source address: This computer has an invalid IP address: 0.0.0.0

### ISSUE THREAD

**Issue by aososam** [solid-software/flutter_vlc_player#255]
When trying to get from network using this code:

`VlcPlayerController _vlcViewController =
      new VlcPlayerController.network("rtsp://192.168.1.9/live/ch00_0");`

`VlcPlayer(
                    controller: _vlcViewController,
                    aspectRatio: 1.23,
                    placeholder: Text("Loading"),
                  )`

On Android it loads fine, but on iOS I get the following error:

"Unable to determine our source address: This computer has an invalid IP address: 0.0.0.0"

**Comment by bachhuberdesign** [comment]
As far as I can tell, this is a MobileVLCKit issue, rather than a flutter_vlc_player issue. We use React Native and are encountering the exact same issue with live555 (which VLC uses for RTSP streaming internally) throwing the invalid IP address error. Some discussion here: https://code.videolan.org/videolan/VLCKit/-/issues/519

**Comment by aososam** [comment]
Yes, it seems like an issue with VLC, not with the Flutter plugin.
I will close this issue and continue in the conversation you mentioned @bachhuberdesign.
